### PR TITLE
Feature/delta transfers

### PR DESCRIPTION
**Pull request by dottedmag**
A part of flexsurfer/re-frisk-remote#3.

### CODE REVIEW

File: re-frisk-sidecar/src/re_frisk_sidecar/client.cljs [hunk 1]
```diff
@@ -41,8 +41,10 @@
         "/chsk" ; Must match server Ring routing URL
         {:type   :auto
          :host   (str "localhost:" js/location.port)
-         :packer (sente-transit/get-transit-packer)})]
-  (def ch-chsk ch-recv)) ; ChannelSocket's receive channel
+         :packer (sente-transit/get-transit-packer)
+         :params {:kind :re-frisk-sidecar}})]
+  (def ch-chsk ch-recv) ; ChannelSocket's receive channel
+  (def chsk-send! send-fn))
```
Comment (flexsurfer) [inline re-frisk-sidecar/src/re_frisk_sidecar/client.cljs]: `chsk-send! ` is not used?

File: re-frisk-sidecar/src/re_frisk_sidecar/core.clj [hunk 2]
```diff
@@ -36,28 +76,63 @@
...
+  (doseq [uid @listeners]
+    (chsk-send! uid [:refrisk/id-handler @id-handler])))
+
+(defmethod -event-msg-handler :refrisk/id-handler-delta
+  [{:keys [?data]}]
+  (swap! id-handler delta/apply ?data)
+  (doseq [uid @listeners]
+    (chsk-send! uid [:refrisk/id-handler @id-handler])))
+
+(defn- get-kind [msg]
```
Comment (flexsurfer) [inline re-frisk-sidecar/src/re_frisk_sidecar/core.clj]: not used?

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: re-frisk-sidecar/src/re_frisk_sidecar/core.clj [hunk 3]
```diff
@@ -36,28 +76,63 @@
...
+  (reset! remote-conn client-id)
+  (when (pos? (count @listeners))
+    (chsk-send! @remote-conn [:refrisk/enable nil])))
+
+(defmethod -event-msg-handler :chsk/uidport-open [{:keys [client-id ring-req]}]
+  (let [kind (get-in ring-req [:params :kind])]
+    (cond
+      (= kind "re-frisk-sidecar") (sidecar-client-connected client-id)
+      (= kind "re-frisk-remote") (remote-connected client-id)
+      :else (println "Connected client of unknown kind" kind))))
```
Comment (flexsurfer) [inline re-frisk-sidecar/src/re_frisk_sidecar/core.clj]: could you please remove this  print

**Comment by flexsurfer** [comment]
thanks @dottedmag , awesome job, lets fix these small comments and squash commits

**Comment by dottedmag** [comment]
Removed unused code and prints, squashed everything here too.

**Pull request merged**